exoplayer-codelab-01/build.gradle

```diff
@@ -1,20 +1,10 @@
-buildscript {
-    repositories {
-        jcenter()
-        google()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.2'
-    }
-}
+apply from: "${project.rootDir}/common.gradle"
 
-allprojects {
-    repositories {
-        jcenter()
-        google()
-    }
-}
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 
-task clean(type: Delete) {
-    delete rootProject.buildDir
-}
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.google.android.exoplayer:exoplayer-core:2.12.0'
```

Comment:
Maybe consider moving `2.12.0` to a gradle variable at the top level, since these are used in each of the modules, and they're all the same version (unless this might change in the future) -- even then, though, it might be useful to have a core, dash, and ui version in that case, so they only have to be updated in one place.

Reply:
Will this affect the user's ability to copy paste? I considered doing this as wel as I do it with the kotlin version, but as the user may be jumping in between modules, I decided to err on the side of copy pasting convenience.

Reply:
I wouldn't get _too_ caught up on copypastability. Maintainability and readability is also important. In this instance I agree with @nic0lette that moving to gradle variables is the better solution since it avoids duplicating the same version string throughout the code, although this could also be done in a follow up PR.

Reply:
Will create an issue as an action item.

Reply:
Created #66